tests/testthat/test-shinytest2-landing_popup.R

```diff
@@ -0,0 +1,184 @@
+testthat::test_that("e2e: teal app with landing_popup_module initializes with no errors", {
+  app <- TealAppDriver$new(
+    data = simple_teal_data(),
+    modules = modules(
+      landing_popup_module(
+        title = "Welcome",
+        content = tags$b("A welcome message!", style = "color: red;")
+      ),
+      example_module()
+    )
+  )
+  app$view()
```

Comment:
This is only for debugging.

```suggestion
```

Reply:
thanks. sure. saw it in some other tests at `test-shinytest2-teal.R` and probably carried over.

Reply:
Yeah, that was my bad. I was focused on helpers that I was committing the test file less cautiously.